mk/xe-linux-distribution

```diff
@@ -135,10 +135,10 @@ identify_redhat()
     # Enterprise Linux Enterprise Linux Server release 5 (Carthage)
     # Enterprise Linux Enterprise Linux Server release 5.5 (Carthage)
     # Oracle Linux Server release 5.6
-    
+
```

Comment:
You text editor removed end of line whitespaces, but this does not belong to this very PR (could be a separate one).

Reply:
@stormi trailing white spaces have been restored